playkit/src/main/java/com/kaltura/playkit/plugins/ads/ima/IMAPlugin.java

```diff
@@ -590,11 +590,11 @@ public void onAdEvent(com.google.ads.interactivemedia.v3.api.AdEvent adEvent) {
                     appInBackgroundDuringAdLoad = true;
                     adsManager.pause();
                 } else {
-                    messageBus.post(new AdEvent(AdEvent.Type.LOADED));
                     if (adPlaybackCancelled) {
```

Comment:
@giladna are you sure `adPlaybackCancelled` will be reset to false correctly if you discard the ad break?
It seems that if it is discarded you won't get to `content_resume_requested` and next time you'll get to loaded the flag will still be up, until you call `resetIMA`